test/com/esotericsoftware/kryo/serializers/GenericsTest.java

```diff
@@ -169,8 +169,7 @@ public void testClassHierarchyWithChangingGenericTypeVariables () {
 		ClassHierarchyWithChangingTypeVariableNames.A<?> o = new ClassHierarchyWithChangingTypeVariableNames.A<>(Enum.class);
 
 		kryo.setRegistrationRequired(false);
-
-		roundTrip(131, o);
+		roundTrip(131, o, false);
```

Comment:
Why did you add `false`? This should pass with both strategies.

Reply:
You're right, removed false and reran test cases